crates/dbs-virtio-devices/src/mmio/mmio_v2.rs

```diff
@@ -316,7 +319,10 @@ where
         // Use mutex for state to protect device.write_config()
         let mut state = self.state();
         if self.check_driver_status(DEVICE_DRIVER, DEVICE_FAILED) {
-            state.get_inner_device_mut().write_config(offset, data);
+            state
+                .get_inner_device_mut()
+                .write_config(offset, data)
+                .unwrap();
```

Comment:
You could not directly use `unwrap()` in non-test area. Could you find a way to handle the error?

Reply:
Fixed.